src/es_archiver/lib/docs/generate_doc_records_stream.js

```diff
@@ -12,6 +12,18 @@ export function createGenerateDocRecordsStream(client, stats) {
         let remainingHits = null;
         let resp = null;
 
+        const alias = await client.indices.getAlias({ index });
```

Comment:
Why not put this into https://github.com/elastic/kibana/blob/master/src/es_archiver/lib/indices/generate_index_records_stream.js#L9?